hw3/01-routing.js

```diff
@@ -1,17 +1,6 @@
 const http = require('http');
-const port = process.env.PORT || 5000;
-
-// http://localhost:5000/welcome should return a status code 200 with a welcome message of your choice in html format
-
-// http://localhost:5000/redirect should redirect the request to '/redirected' by using 302 as the status code / the redirected page should return a redirected message of your choice
-
-// http://localhost:5000/cache should return 'this resource was cached' in html format and set the cache max age to a day
-
-// http://localhost:5000/cookie should return 'cookies… yummm' in plain text and set 'hello=world' as a cookie
 
-// http://localhost:5000/check-cookies should return 'yes' / 'no' in plain text depending on whether the browser has the 'hello' cookie
-
-// For other routes, such as http://localhost:5000/other, this exercise should return a status code 404 with '404 - page not found' in html format
+const port = process.env.PORT || 5000;
```

Comment:
Can you update this to port 5001? I'm getting a 'port already in use error' because it's still 5000. Your 01-routing.js and 02-url.js files are using 5000 while your 03-form.js file is using 5001.

Reply:
you will get the port already in use error if you are running 01 and 02 together or either of them twice or another process which is listening on that port.

I will change them to 5001,5002,5003, so that you can run them at the same time while testing, though.